mantatail.py

```diff
@@ -237,29 +334,38 @@ def __init__(self, channel_name: str, user: UserConnection) -> None:
         self.set_operator(user)
 
     def set_operator(self, user: UserConnection) -> None:
+        """Adds a user to the channel's operators."""
         self.operators.add(user.nick.lower())
 
     def remove_operator(self, user: UserConnection) -> None:
+        """Removes a user from the channel's operators."""
         self.operators.discard(user.nick.lower())
 
     def is_operator(self, user: UserConnection) -> bool:
+        """Checks if a user is an operator on the channel."""
         return user.nick.lower() in self.operators
 
     def kick_user(self, kicker: UserConnection, user_to_kick: UserConnection, message: str) -> None:
+        """
+        Notifies all users on the channel that a user has been kicked.
+        Thereafter removes the kicked user from the channel.
+        """
         for usr in self.users:
             usr.send_que.put((message, kicker.get_user_mask()))
 
         self.users.discard(user_to_kick)
 
 
 def split_on_new_line(string: str) -> List[str]:
+    """Splits a message received by a client on "\r\n" (IRC) or "\n" (Netcat)."""
```

Comment:
I don't like `(IRC)` and `(Netcat)`

```suggestion
    """Splits a message received by a client on "\r\n" (most IRC clients) or "\n" (e.g. Netcat)."""
```